app/controllers/StoryPackagesController.scala

```diff
@@ -66,6 +69,15 @@ object StoryPackagesController extends Controller with PanDomainAuthActions {
   }
 
   def deletePackage(id: String) = APIAuthAction.async { request =>
-    Database.removePackage(id).map(_ => Ok)
+    for {
+      storyPackage <- Database.getPackage(id)
+    } {
+      val identity = request.user
+      UpdatesStream.putStreamUpdate(StreamUpdate(DeletePackage(id), identity.email, Map(id -> CollectionJson(
+        List(), None, None, DateTime.now, "", "", None, None, None)),
+      storyPackage, true))
+    }
+
+      Database.removePackage(id).map(_ => Ok)
```

Comment:
`UpdatesStream.putStreamUpdate` should be done only after the package is correctly removed from database. You don't have to query `Database.getPackage`, you can probably get the story package from `removePackage` if you call getDeleteItemResult`